services/dm_service.py

```diff
@@ -4,7 +4,9 @@
 from helpers import programmes_helper
 from datetime import datetime, date
 import re
-
+import sys
+sys.path.insert(1,'/path/to/application/app/folder')
```

Comment:
I'm not entirely sure what you are doing here, can you just import the year instead?

Reply:
Actually, in the previous PR this was the reason I had made another file constant.py over there, to keep things simple, since the files where we have to import year are inside one directory and the constants.py you had was one step above in the tree, so we had to use sys to get the path in order to access the file that is not in the current directory, because by default python looks only in the current directory😅

Reply:
We don't need this since this module is imported by `main.py`, which is in the outer directory, and this allows it to access modules in the outer directory.

Reply:
oh.. I didn't know, so how exactly should I import this then..

Reply:
because, removing those two lines alone might not fix the issue right, it shows me 'no such module found'

Reply:
`import constants` should be enough! See [here](https://github.com/martinmladenov/RankingBot/blob/4eb25cb99c821e82f4b47dc36447556a8f9a676d/commands/sendreminderdm_command.py#L6) for an example.

Reply:
Wow, thanks!

Reply:
Did the same way in the rest of the files too

Reply:
<img width="367" alt="check" src="https://user-images.githubusercontent.com/56349099/96603169-641e2100-1311-11eb-86da-c8d5970dd879.PNG">
For some reason shows like this to me as I run the code, that's why I was looking for other ways😅

Reply:
That's because the module which you execute is in the services directory. If you were to run `main.py` (which would require some additional configuration), you wouldn't get this error.

Reply:
Thanks for letting me know, I didn't try running main.py since I didn't have some modules like discord installed in my system. So is everything fine now?